### PR TITLE
Add ws-fancy-currency and ws-fancy-percent directives

### PR DESCRIPTION
**Pull request by 6**
screen shot 2016-06-16 at 3 19 54 pm

### CODE REVIEW

File: src/formatting/fancy-currency/ws-fancy-currency.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+(function () {
+  'use strict';
+
+  angular
+  .module('ws.common')
+  .directive('wsFancyCurrency', ['$templateCache', '$sce', function ($templateCache, $sce) {
```
Comment (DanGodfrey) [inline src/formatting/fancy-currency/ws-fancy-currency.js]: unused dependency $sce

**Comment by DanGodfrey** [comment]
looking good! I'll give the tests a quick once over when they're ready.

**Comment by 6** [comment]
fixed! please re-review @DanGodfrey

**Comment by DanGodfrey** [comment]
:shipit:

**Pull request merged**